build/compiler.js

```diff
@@ -608,8 +603,7 @@ function Compiler({
       },
       reporter: null,
       serverSideRender: true,
-      // TODO(#14): handle route prefix here??
-      publicPath: '/_static/',
+      publicPath: '/_static', // keep in sync with defaults in environment-variables-plugin.js
```

Comment:
Only relevant change in this file.  Remainder is `prettier`'s doing.

Reply:
Weird, linters are failing. Do you have an older prettier version installed?